pkg/database/vercode.go

```diff
@@ -106,16 +106,27 @@ func (v *VerificationCode) BeforeSave(tx *gorm.DB) error {
 // to update statistics about usage. If the executions fail, an error is logged
 // but the transaction continues. This is called automatically by gorm.
 func (v *VerificationCode) AfterCreate(scope *gorm.Scope) {
+	v.updateStats(scope, true)
+}
+
+func (v *VerificationCode) updateStats(scope *gorm.Scope, increment bool) {
 	date := timeutils.Midnight(v.CreatedAt)
 
+	var incString string
```

Comment:
simplification

```
sign := "+"
if !increment {
  sign = "-"
}
```

And then the 1 can be embedded in the statement still

Reply:
sure